<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpSideBar</class>
 <widget class="QDockWidget" name="HelpSideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&amp;Contents</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabContents">
       <attribute name="title">
        <string>&amp;Contents</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QTreeView" name="listView">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSearching">
       <attribute name="title">
        <string>Se&amp;arch</string>
       </attribute>
       <layout class="QVBoxLayout" name="_2">
        <item>
         <layout class="QHBoxLayout" name="_3">
          <item>
           <widget class="QLineEdit" name="searchingEdit">
            <property name="toolTip">
             <string>Searching is case insensitive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchingButton">
            <property name="text">
             <string>&amp;Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="searchingView">
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBookmarks">
       <attribute name="title">
        <string>Book&amp;marks</string>
       </attribute>
       <layout class="QVBoxLayout" name="_4">
        <item>
         <widget class="QTreeWidget" name="bookmarksView">
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_5">
          <item>
           <widget class="QPushButton" name="bookmarkButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
            <property name="icon">
             <iconset theme="list-add"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBookmarkButton">
            <property name="text">
             <string>&amp;Remove</string>
            </property>
            <property name="icon">
             <iconset theme="list-remove"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteAllBookmarkButton">
            <property name="text">
             <string>R&amp;emove All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
